recipes/zlib-ng/all/conanfile.py

```diff
@@ -0,0 +1,114 @@
+from conans import ConanFile, CMake
+from conans.tools import download, unzip, replace_in_file, remove_files_by_mask, get
+import os
+
+class ZlibNgConan(ConanFile):
+    name = "zlib-ng"
+    description = "zlib data compression library for the next generation systems"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage="https://github.com/Dead2/zlib-ng/"
+    license="https://github.com/Dead2/zlib-ng/blob/ba7f0eb6e294306ac6d771216ea4442f2ea2d830/LICENSE.md"
+    topics = ("conan", "zlib", "compression")
+    exports_sources = ["CMakeLists.txt"]
+    generators = "cmake", "cmake_find_package"
+    settings = "os", "arch", "compiler", "build_type"
+    options = {"shared": [True, False]}
+    default_options = "shared=False"
```

Comment:
Add fPIC + use dicts
```suggestion
    options = {"shared": [True, False], "fPIC": [True, False]}
    default_options = ["shared": False, "fPIC": True}
```